docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.yml

```diff
@@ -0,0 +1,92 @@
+
+          - `makeEwsRequestAsync`
+          - `getUserIdentityTokenAsync`
+          - `getCallbackTokenAsync`
+
+          If your add-in calls any of these APIs, you should adopt NAA and migrate to using Entra ID tokens to access Microsoft Graph instead.
+      - question: Which Outlook add-ins are in scope?
+        answer: |
+          Many of our most important add-ins are in scope. If your add-in is using EWS or Outlook REST to access Exchange Online resources, it almost certainly needs to migrate off of legacy Outlook tokens to NAA.
+          If your add-in is for Exchange on-premises only (for example, Exchange 2019), it is not affected by this announcement.
+      - question: What will happen to my Outlook add-ins if I don't migrate to NAA?
+        answer: |
+          If you don't migrate your Outlook add-ins to NAA, they will stop working as expected in Exchange Online. Beginning in April 2024, Exchange Online admins will have the abilitybe able to block legacy token issuance in their tenants. In October 2024, Exchange Online will block legacy token issuance by default in all tenants. Any add-in that uses legacy tokens will not be able to access Exchange online resources.
+          If your add-in only works on-premises or if your add-in is on a deprecation path, you may not need to update. However, most add-ins that access Exchange resources through EWS or Outlook REST will have to migrate to continue functioning as expected.
+      - question: How do I migrate my Outlook add-ins to NAA?
+        answer: |
+          To support NAA in your Outlook add-in, please refer to the following documentation and sample.
+
+          - [Enable SSO in an Office Add-in using nested app authentication](enable-nested-app-authentication-in-your-add-in.md).
+          - [Outlook add-in with SSO using nested app authentication](https://github.com/OfficeDev/Office-Add-in-samples).
+      - question: As an admin, how do I control legacy token issuance in my Exchange Online tenant?
+        answer: |
+          To keep things simple for Exchange admins the token deprecation process is similar to what admins did for Basic Auth. You can reuse your Basic Auth playbook in Exchange to control Exchange legacy token issuance — they even use the exact same Exchange cmdlet.
+          To control legacy token issuance, use the `Set-AuthenticationPolicy` cmdlet with the `-AllowLegacyExchangeTokens` parameter to create or update an Exchange authentication policy that blocks issuance of Exchange user identity tokens and callback tokens. Once you have created or updated the authentication policy, you can assign it to test mailboxes or to your entire organization. We recommend deploying any new policies to a set of test mailboxes to make sure you don’t have unintended effects on add-ins your organization needs.
```

Comment:
```suggestion
```
Unfortunately this is not ready yet -- we should add this back in in ththe coming weeks